### ISSUE TITLE
Site core web vitals have no labels

### ISSUE THREAD

**Issue by noah-jk** [kwall-llc/Accorbis-web#1]
<p>The core web vitals panel have the score, but they're not labeled (performance, accessibility etc.) </p>
<img width="1520" alt="Image" src="https://github.com/user-attachments/assets/234e70f1-bc6c-4058-921e-9f60f42a05de" />
